File: arrays 101/sort_array_by_parity.py
class Solution:
    def sortArrayByParity(self, A: List[int]) -> List[int]:
        B = []
        for i in A:
            if i% 2 == 0:
                B.append(i)
        for i in A:
            if i%2 != 0:
                B.append(i)
        return B

File: arrays 101/sort_array_by_parity.java
class Solution {
    public int[] sortArrayByParity(int[] A) {
        
        int[] B = new int[A.length];
        int len = 0;
        int len2 = A.length - 1;
        for(int i = 0; i<A.length; i++)
        {
            if (A[i] % 2 == 0)
            {
                B[len++] = A[i];
            }
            
            else
            {
                B[len2--] = A[i];
            }
        }
        
        return B;
        
    }
}